MediatorPattern/Tester.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace MediatorPattern
+{
+    class Tester : Colleague
+    {
+        public Tester(Mediator mediator) : base(mediator) {}
+
+        public override void Notify(string message)
+        {
+            Console.WriteLine($"Message to tester: {message}");
```

Comment:
Can we have something else instead of a Console.WriteLine, which can help people understand the necessity of the Mediator Pattern?

Reply:
Suggest something, please)

Reply:
I am planning on re-writing most of the code sometime soon. Will ping all individual pattern contributors to get their input when I do it.

Reply:
Ok, thank you!